projects/auth0-angular/src/lib/auth.config.ts

```diff
@@ -265,7 +265,7 @@ export class AuthClientConfig {
   /**
    * Gets the config that has been set by other consumers of the service
    */
-  get(): AuthConfig {
+  get(): AuthConfig | undefined {
```

Comment:
Even tho I understand that this change is, strictly speaking, better. However, this is a breaking chance and I would like to avoid doing so for now. Can we revert this change so that it works in strict mode without changing the return type?

Reply:
yes, we could do type assertion like this
```
get(): AuthConfig {
    return this.config as AuthConfig;
}
```
Or we could change the `config` field declaration to `private config!: AuthConfig;`
Looks like the field is optional here, so the former is better.
What do you think?

Reply:
I've just found one more option
In `Auth0ClientFactory.createClient` we have
```
    const config = configFactory.get();

    if (!config) {
      throw new Error(
        'Configuration must be specified either through AuthModule.forRoot or through AuthClientConfig.set'
      );
    }
```

We could move Error thorwing into `configFactory.get()` method.
But in this case we could potentially affect the `configFactory.get()`'s callers

Reply:
Looks like the field is optional here, so the option 1 is better. Plus we would not affect the callers

Reply:
Yes 1 is fine. I don't think we should be moving anything only for the purpose of fixing strict types.

Reply:
added type assertion